pom.xml

```diff
@@ -48,18 +48,18 @@
         <netty.version>4.1.51.Final</netty.version>
         <netty.boringssl.version>2.0.34.Final</netty.boringssl.version>
         <junit.version>4.13.1</junit.version>
-        <mockito.version>1.10.19</mockito.version>
+        <mockito.version>2.0.2-beta</mockito.version>
```

Comment:
why do we need to update to a bet version ?
is there a real benefit ?
otherwise I am -1 here

Reply:
Okay, I downgraded to the latest stable